ga4gh/datamodel/variants.py

```diff
@@ -453,6 +464,11 @@ def getVariants(self, referenceName, startPosition, endPosition,
         # protocol version 0.6
         if callSetIds is None:
             callSetIds = []
+        else:
+            for callSetId in callSetIds:
+                if callSetId not in self._callSetIds:
+                    raise exceptions.CallSetNotInVariantSetException(
+                        callSetId, self.getId())
```

Comment:
fixed a bug here. was calling self._varientset. which was not actually an attribute of this class.